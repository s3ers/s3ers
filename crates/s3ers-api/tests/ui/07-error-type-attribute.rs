use s3ers_api::s3ers_api;

s3ers_api! {
    metadata: {
        description: "Does something.",
        method: POST, // An `http::Method` constant. No imports required.
        name: "some_endpoint",
        path: "/some/endpoint/:baz",
        authentication: None,
    }

    request: {}

    response: {}

    #[derive(Default)]
    error: s3ers_api::error::SError
}

fn main() {}
