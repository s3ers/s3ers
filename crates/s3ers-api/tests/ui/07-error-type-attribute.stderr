error: unexpected token
  --> $DIR/07-error-type-attribute.rs:16:5
   |
16 |     #[derive(Default)]
   |     ^
